tasks/tasks-linear-fd05f8a0-420a-4313-a462-690881c0a45c.md

```
@@ -0,0 +1,82 @@
+- [x] `remove fallback`
+- [x] `expire fallback`
+- [x] `justify retaining fallback`
+- [x] `CO-382`
+- [x] `CO-394`
+- [x] `CO-395`
+- [x] `CO-396`
+- [x] `CO-397`
+- [x] `CO-398`
+
+## CO-382 Fallback Decision Table
+| Surface | Fallback / seam | Decision | Owner | Trigger | Introduced date | Review date | Maximum lifetime | Removal condition | Validation |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)